build_engine_test_images/scripts/cleanup.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+if [[ ! -v TF_VAR_tf_workspace ]]; then
+	export TF_VAR_tf_workspace="$(dirname $(dirname -- $(readlink -fn -- "$0")))"
```

Comment:
Could we pass TF_VAR_tf_workspace into the container as an environment variable and keep it always the same throughout the container lifecycle?

Reply:
Yes I modified. set TF_VAR_tf_workspace as container env var below 
```
sh """ docker run -itd --name ${JOB_BASE_NAME}-${BUILD_NUMBER} \
                                   --env TF_VAR_build_engine_aws_access_key=${AWS_ACCESS_KEY} \
                                   --env TF_VAR_build_engine_aws_secret_key=${AWS_SECRET_KEY} \
                                   --env TF_VAR_docker_id=${DOCKER_USERNAME} \
                                   --env TF_VAR_docker_password=${DOCKER_PASSWORD} \
                                   --env TF_VAR_docker_repo=${DOCKER_REPO} \
                                   --env TF_VAR_tf_workspace=${TF_VAR_tf_workspace} \
                                   ${imageName}
            """

```